docs/MSPX.md

```diff
@@ -19,5 +17,15 @@ Further parameter settings:
 - "Rx Snd RcChannel": When set to "rc override" or "rc channels", SET_RAW_RC MSP messages are sent to the flight controller. In the flight controller configure the receiver to the MSP protocol. This allows one to avoid the extra wire for CRSF.
 - "Rx Snd RadioStat": has no effect.
 
+## INAV Configuration
+
+To use a mLRS Receiver with INAV in MspX mode, the following settings have to be applied:
+- Enable MSP for the Serial Port the mLRS receiver is conencted to (do not enable Serial RX)
+- Set the Baud-Rate to 115200 or higher (Only INAV 7.1 and 8.0-dev are tested, Version 6.0 and newer should also work)
+- INAV 8.0 and later will also support Baud 230400 but with no noticeable difference in performance or stability 
+![image](https://github.com/user-attachments/assets/c359f819-66dd-453c-8883-f2498103c7a6)
```

Comment:
Could you add a comment something like 'An example for UART2 is shown below' and/or maybe highlight the row with a box?